libs/ui/src/system_administrator_screen_contents.test.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import { fakeKiosk, mockOf } from '@votingworks/test-utils';
+import { fakeLogger } from '@votingworks/logging';
+import { render, screen } from '@testing-library/react';
+import { usbstick } from '@votingworks/utils';
+
+import { isVxDev } from './config/features';
+import { SystemAdministratorScreenContents } from './system_administrator_screen_contents';
+
+jest.mock('./config/features', (): typeof import('./config/features') => {
+  return {
+    ...jest.requireActual('./config/features'),
+    isVxDev: jest.fn(),
+  };
+});
+
+beforeEach(() => {
+  mockOf(isVxDev).mockImplementation(() => false);
```

Comment:
this is fine, though I think the right mock here is not the guts of what turns on the feature (vxdev or not is a bit specific), and more a mock of just having a particular feature enabled, whatever the mechanism. I think worth fixing when we settle on whether we want to require vxdev or not.

Reply:
Ah! I _think_ there may be a misunderstanding here. None of this PR is actually gated behind the VVSG2 auth flows feature flag. Once I merge this, regardless of the status of the VVSG2 auth flows feature flag, the new screens in this PR will be visible. (I didn't see a need to gate them.)

What _is_ gated and will in fact be permanently gated by VxDev or not is the VxDev-only "Quit" button. This mocking is to control the display and testing of that button.

Correct me if I'm missing something of course!

Reply:
gotcha! Makes sense. Carry on :)